backend/main.go

```diff
@@ -334,16 +414,19 @@ func (s *Server) handleImprove(w http.ResponseWriter, r *http.Request) {
 func main() {
 	var useInMemStore bool
 	var authMode string
-	var authDomain string
+	var defaultDomain string
 	flag.BoolVar(&useInMemStore, "inmemstore", false, "Use in-memory datastore")
 	flag.StringVar(&authMode, "authmode", "none", "Set authentication mode, either 'none' or 'firebase'")
-	flag.StringVar(&authDomain, "authdomain", "google.com", "The email domain with read and write access")
+	flag.StringVar(&defaultDomain, "defaultdomain", "none", "The domain that all team permissions are defaulted to")
```

Comment:
If this only affects in-mem store, we should probably say so in the docstring.

Reply:
👍🏻